locales/en.json

```diff
@@ -1462,7 +1462,7 @@
 		"label": {
 			"newPassword": "New password:",
 			"repeatYourNewPassword": "Repeat your new password:",
-			"under14YearsOld": "under 14 years old."
+			"under14YearsOld": "13 years or younger."
```

Comment:
please change key to under13YearsOld

Reply:
key should stay like this as under 14 means 13 or younger

Reply:
ah yes, my mistake ;)